tests/pyautomagic/test_Project.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from pyautomagic.src import Block, Project, Subject, Config
+
+name = 'Stupid project'
+d_folder = r"C:\Users\saul__000\OneDrive\Escritorio\Johns Hopkins\NeuroData Design I\Project_Test_Folder"
+ext = 'something.edf'
+
+
+def test_data_folder():
+    test_Project = Project.Project(name, d_folder, ext)
+    assert(test_Project.data_folder == r"C:\Users\saul__000\OneDrive\Escritorio\Johns Hopkins\NeuroData Design I\Project_Test_Folder")
+
+
+def test_results_folder():
+    test_Project = Project.Project(name, d_folder, ext)
+    assert(test_Project.results_folder == r"C:\Users\saul__000\OneDrive\Escritorio\Johns Hopkins\NeuroData Design I\Project_Test_Folder\derivatives\automagic")
+
+
+def test_file_extension():
+    test_Project = Project.Project(name, d_folder, ext)
+    assert(test_Project.file_extension == ".edf")
+
```

Comment:
these will all fail as soon as this isn't on your computer anymore

Reply:
> these will all fail as soon as this isn't on your computer anymore

I'm having some trouble while testing the results folder without using my local paths. How can I achieve that? The functions that I have to look for the data folder and to set the results folder only works if the specified folder exists in the computer, so how can I test those functions without using my local data folders?

Reply:
I'm having some trouble while testing the results folder without using my local paths. How can I achieve that? The functions that I have to look for the data folder and to set the results folder only works if the specified folder exists in the computer, so how can I test those functions without using my local data folders?

Reply:
You can look at my PR for how to make this work.